static/js/types/types.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { expectTypeOf } from "expect-type"
```

Comment:
I've used this in the past on some personal projects and when contributing typedefs to some open source repos (e.g.,  https://github.com/sequelize/sequelize). It's nice if you want to assert stuff about types in tests.